addon/services/-gestures.js

```diff
@@ -44,7 +38,6 @@ export default Service.extend({
   },
 
   setupRecognizer(name, details) {
-    if (this.get('_fastboot.isFastBoot')) { return; }
```

Comment:
@anthonybalmeo - sorry - u mean this check did the job

that's odd but I buy it, i didn't read the code that closely and have no way to play with it unless i install the addon, etc - if moving the check here is all we need feel free to add it and drop me a sha, i'll cherry pick

(thanks, sorry for the confusion)